src/ngx_http_small_light_imagemagick.c

```diff
@@ -233,6 +236,24 @@ ngx_int_t ngx_http_small_light_imagemagick_process(ngx_http_request_t *r, ngx_ht
             DestroyString(of_orig);
             return NGX_ERROR;
         }
+
```

Comment:
In `ngx_small_light`, 4-space indentation is recommended instead of 2-space's. And is this block not be required when `rmprof=y`?